src/null-safety/index.md

```diff
@@ -344,6 +344,12 @@ The Dart team is currently aware of the following issues:
   * Migration of the pub.dev packages owned by the Dart team
     is nearly complete, but a few are still missing. See pub.dev for
     [null-safe packages from the Dart team][ns-dart-pkgs].
+  * When publishing packages with a regular (non-dev) dependency on
+    the `test` package you may get pub warnings about it not being
+    fully migrated. As long as you are not ignoring any of the
+    `import_of_legacy_library_into_null_safe` or
```

Comment:
Can we add links to those lints?

Reply:
I would if I had any idea where to link to :)

Reply:
I'll get this...

Reply:
As a reference I don't know if there is currently on for `import_of_legacy_library_into_null_safe` but there is one for [`export_legacy_symbol`](https://dart.dev/tools/diagnostic-messages#export_legacy_symbol).